lib/tasks/coverage_report.rake

```diff
@@ -7,16 +7,19 @@ namespace :coverage do
   desc "Collates all result sets generated by the different test runners"
   task :report do
     require 'simplecov'
+    require 'fileutils'
 
     SimpleCov.coverage_dir('coverage_results')
 
     if ENV['CI']
-      SimpleCov.collate(Dir["*/coverage_*/.resultset.json"]) do
+      SimpleCov.collate(Dir['*/coverage_*/.resultset.json']) do
         refuse_coverage_drop
       end
     else
-      SimpleCov.collate(Dir["lib/coverage_*/.resultset.json"])
+      SimpleCov.collate(Dir['lib/coverage_*/.resultset.json'])
     end
+
+    Dir['lib/coverage_{[!r][!e][!s][!u][!l][!t][!s]}*'].each { |dir| FileUtils.rm_rf(dir) }
```

Comment:
r-e-s-u-l-t-s, find out what they mean to us
r-e-s-u-l-t-s, take care of directories

sock 'em to us, sock 'em to us, sock 'em to us, sock 'em to us
a few results
sock 'em to us, sock 'em to us, sock 'em to us, sock 'em to us
whoa, devs